### PR TITLE
Playlist UI XML created

### PR DESCRIPTION
**Pull request by vinodghai**
resolves https://github.com/brave-experiments/playlist-kotlin/issues/1

### CODE REVIEW

File: src/main/res/values/strings.xml [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    <string name="remove_data">Remove Data</string>
+    <string name="number_selected">%d Selected</string>
+    <string name="number_items">%d items</string>
+    <string name="play">Play</string>
+    <string name="shuffle">Shuffle</string>
+    <string name="recently_played">Recently played</string>
+    <string name="playlist_next">Next</string>
+    <string name="tap_long_press_or_drag">Tap, long press or drag</string>
+    <string name="add_to_playlist">Add to Playlist</string>
+    <string name="string_playlists">Playlists</string>
```
Comment (SergeyZhukovsky) [inline src/main/res/values/strings.xml:26]: How are we going to translate above strings? Perhaps it's better to pull strings from the main application when we init that library?

**Comment by SergeyZhukovsky** [comment]
We want to add Brave's headers as well to every file.

**Comment by SergeyZhukovsky** [comment]
@vinodghai have you tried to link it to the browser? Do you have any screenshots of how does it look like?

**Comment by deeppandya** [comment]
Closing this PR as created a new one.

**Pull request closed without merging**